src/content/pages/community-activities.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: Community Organisers Activities
+url: /community-activities
+subtitle:
+  Booths, Open spaces, and more activities for organisers
+---
+
+# Community Organisers Activities
+
+The Python community is an essential part of the language, and for many people
+the reason for sticking around and keep meetups, conferences, forums, and many
+more things running, in order to help others.
+
+This year at EuroPython, we have a few activities focused on some of the
+communities around Europe and world, but also initiatives around Python.
+
+
+## Community Organisers Open Space
+
+Communication is the key to avoid repetition and failing on the same mistakes.
+Let's be together and share the experience of our communities and our personal
+challenges and wins so more communities can improve.
+
+This year, we are saving a two-hour slot in the Open Spaces for all the community
+and conference organisers that want to share their experiences, and more
+importantly, to see how the EuroPython Society can help you to achieve your
+goals and solve your current issues.
```

Comment:
```suggestion
Communication is key to avoiding repetition and learning from past mistakes.
Let's come together and share the experiences of our communities, as well as and our personal
challenges and successes, so that more communities can grow and improve.

This year, we are dedicating a two-hour slot in the Open Spaces to all community organisers who want to share their experiences. More
importantly, it's a chance to explore how the EuroPython Society can support you in achieving your
goals and addressing your current challenges.
```